### ISSUE TITLE
Android Support?

### ISSUE THREAD

**Issue by DrSouil** [keijiro/MidiJack#18]
Hi, I'm working on a google cardboard music game. It would be perfect if it can support Android

**Comment by keijiro** [comment]
I have no idea about MIDI support on Android. By the way, this is just out of curious but do you know any good Android app that supports MIDI devices? I wonder if it's actually possible.

**Comment by DrSouil** [comment]
Yes , My MIDI keyboard is XKEY and the Android app is WALK BAND. I connect my keyboard to my phone through an OTG cable and it works fine. Infact there are APIs about midi in Android :https://developer.android.com/reference/android/media/midi/package-summary.html. And this answer about the connection may help:http://answers.unity3d.com/questions/679985/access-android-usb-as-host-otg.html.

**Comment by keijiro** [comment]
Thanks very much for the information. That's informative.

**Comment by momo-the-monster** [comment]
I have a working Android MIDI plugin for Unity and I use MidiJack for all my MIDI projects. I'll make it compatible with MidiJack and issue a pull request when it's ready.

**Comment by 01GOD** [comment]
Sounds good, thanks. I was wondering about that also. Have only used mobile
midi on IOS.

**Comment by momo-the-monster** [comment]
Basic Android support is included in this pull request: #19 
There's some additional refactoring and cleanup that could be done, but since this is working, I thought I'd submit now so the interested parties can start using it.

**Comment by 01GOD** [comment]
Somehow had missed post until today. Thanks!
